payments-core/src/main/java/com/stripe/android/IntentConfirmationInterceptor.kt

```diff
@@ -0,0 +1,226 @@
+package com.stripe.android
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.core.injection.PUBLISHABLE_KEY
+import com.stripe.android.core.injection.STRIPE_ACCOUNT_ID
+import com.stripe.android.core.networking.ApiRequest
+import com.stripe.android.model.ConfirmPaymentIntentParams
+import com.stripe.android.model.ConfirmPaymentIntentParams.SetupFutureUsage.OffSession
+import com.stripe.android.model.ConfirmStripeIntentParams
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.android.model.StripeIntent
+import com.stripe.android.networking.StripeRepository
+import javax.inject.Inject
+import javax.inject.Named
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface IntentConfirmationInterceptor {
```

Comment:
Do we expect merchants to implement this interface? Do we expect this to be public when we GA?

Reply:
Nope, that’s going to stay internal and be used in Link and PaymentSheet.